web/src/app/shared/models/profile.model.ts

```diff
@@ -33,5 +33,5 @@ export class ProfileModel {
   repositories?: RepositoriesModel = new RepositoriesModel();
   activity?: ActivityModel[] = [];
   following?: string[] = [];
-  isAdmin?: Boolean = false;
+  isAdmin: boolean = false;
```

Comment:
Making it required creates an issue and it throws an error
`Type 'ProfileModel' is not assignable to type '{ uid: string; username: string; name: string; email: string; phone: string; avatarUrl: string; oauth: { githubToken: any; token: string; expirationTime: string; authTime: string; issuedAtTime: string; signInProvider: string; claims: { ...; }; }; emailVerified: boolean; creationTime: string; lastSignInTime: string; }'.
    Property 'oauth' is optional in type 'ProfileModel' but required in type '{ uid: string; username: string; name: string; email: string;
phone: string; avatarUrl: string; oauth: { githubToken: any; token: string; expirationTime: string; authTime: string; issuedAtTime: string;
signInProvider: string; claims: { ...; }; }; emailVerified: boolean; creationTime: string; lastSignInTime: string; }'.`